vm-validator/src/vm_validator.rs

```diff
@@ -88,6 +88,10 @@ impl<S: StateView> ValidationState<S> {
         self.state_view = state_view;
     }
 
+    fn state_view_id(&self) -> StateViewId {
+        self.state_view.id()
```

Comment:
Should we make this method non-default? Seems error prone to end up with misc state view id

Reply:
probably, but there's too many of them 🤦‍♂️